src/managers/epicenter-channel-manager.js

```diff
@@ -182,6 +182,35 @@ var EpicenterChannelManager = classFrom(ChannelManager, {
         return __super.getChannel.call(this, { base: baseTopic });
     },
 
+    /**
+     * Create and return a publish/subscribe channel (from the underlying [Channel Manager](../channel-manager/)) that automatically tracks the presence of an [end user](../../../glossary/#user), that is, whether the end user is currently online in this group and world. Notifications are automatically sent when the end user comes online, and when the end user goes offline (not present for more than 2 minutes). Useful in multiplayer games for letting each end user know whether other users in their shared world are also online. 
+     *
+     * **Example**
+     *
+     *     var cm = new F.manager.ChannelManager();
+     *     var worldManager = new F.manager.WorldManager({
+     *         account: 'acme-simulations',
+     *         project: 'supply-chain-game',
+     *         model: 'model.eqn'
+     *     });
+     *     worldManager.getCurrentWorld().then(function (worldObject, worldService) {
+     *         var presenceChannel = cm.getPresenceChannel(worldObject);
+     *         presenceChannel.on('presence', function (evt, notification) {
+     *              console.log('online', notification);
```

Comment:
`notification` is actually an object with `{ userId: incomingUserId, online: boolan }`